lib/associations/belongs-to-many.d.ts

```diff
@@ -4,7 +4,7 @@ import {
   ManyToManyOptions,
   AssociationScope,
   ForeignKeyOptions,
-  MultiAssociationAccessors
+  MultiAssociationAccessors,
```

Comment:
revert

Reply:
Might wanna add tslint to your CI 😛

Reply:
while yes, more TSLint rules would be good, you should never make changes to a codebase that are not in the scope of the PR.